.eslintrc.js

```diff
@@ -4,18 +4,29 @@ module.exports = {
 		jest: true,
 		browser: true,
 	},
-	extends: ['@guardian/eslint-config-typescript'],
```

Comment:
I think we still want to extend from the base config, [as per the docs](https://github.com/guardian/csnx/tree/main/libs/@guardian/eslint-config-typescript#usage).

Reply:
We're extending it for all the `'.ts','*.tsx'` and using @guardian/eslint-config for `'.js','*.jsx'` in the overrides.

Reply:
Why is that?

Reply:
To make sure it's only applied to .ts files. When i use this to extend all files. I get this error:  

`
/code/consent-management-platform/jest.config.js
  4:13  error  Require statement not part of import statement  @typescript-eslint/no-var-requires
  `

Reply:
Thansk, I found a way to resolve this.